src/DurableTask.SqlServer/Scripts/schema-1.2.0.sql

```diff
@@ -0,0 +1,33 @@
+﻿-- Copyright (c) Microsoft Corporation.
+-- Licensed under the MIT License.
+
+-- PERSISTENT SCHEMA OBJECTS (tables, indexes, types, etc.)
+--
+-- The contents of this file must never be changed after
+-- being published. Any schema changes must be done in
+-- new schema-{major}.{minor}.{patch}.sql scripts.
+
+-- Add a new TraceContext column to the Instances table
+IF NOT EXISTS (SELECT 1 FROM sys.columns WHERE object_id = OBJECT_ID('__SchemaNamePlaceholder__.Instances') AND name = 'TraceContext')
+    ALTER TABLE __SchemaNamePlaceholder__.Instances ADD TraceContext varchar(800) NULL
```

Comment:
This is the first release where we have a database schema upgrade! Schema upgrades must be done by adding a new `schema-X.Y.Z.sql` file, and each file only contains changes, like adding columns.

Reply:
Yes, it's very similar to the Antares database upgrade model.

Reply:
I also added a new README.md file in the latest commit to explain the process in more detail. That will probably be easier than studying old PRs (though that will still be an option).